samples/ZigBeeNet.PlayGround/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "ZigBeeNet.PlayGround": {
       "commandName": "Project",
-      "commandLineArgs": "--zbd 2"
+      "commandLineArgs": "--zbd 1 --p COM5 --b 9600"
```

Comment:
avoid hardcode com port

Reply:
this has been committed by mistake, I will revert